compute/instances/troubleshooting/fix-broken-vpn-when-switching-openvpn-vm-to-routed-ip.mdx

```diff
@@ -0,0 +1,57 @@
+---
+meta:
+  title: Fix broken VPN when switching OpenVPN Instance to routed IP
+  description: This page helps you recover VPN functionality when switching your OpenVPN instance to routed IP
+content:
+  h1: Fix broken VPN when switching OpenVPN Instance to routed IP
+  paragraph: This page helps you recover VPN functionality when switching your OpenVPN instance to routed IP
+tags: openvpn  instance routed ip
+dates:
+  validation: 2024-10-11
+  posted: 2024-10-11
+categories:
+  - compute
+---
+
+When switching an OpenVPN Instance to routed IP, the configured VPN will fail to work after the reboot. This is caused by a bug in a script that has been fixed on newer versions of the OpenVPN image.
+
+This can now be fixed by installing a new version of the OpenVPN scripts which has been packaged Scaleway's stable PPA that contained the fixed script.
```

Comment:
There's a preposition missing here somewhere but I can't work out where. "into which has been packaged"? or "which contains a package for Scaleway's stable bla bla"?

Reply:
```suggestion
This can now be fixed by installing a new version of the OpenVPN scripts, which has been packaged in Scaleway's stable PPA and contains the fixed script.
```